src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -64,6 +62,9 @@
       @end
 
       {@class.requestBodyTypeName} {@class.requestBodyVarName} = new {@class.requestBodyTypeName}();
+      @join field : class.requestBodyFields if class.requestBodyFields
```

Comment:
Isn't this more properly written as an `@if` clause with `@join` inside, like the one added above? (I had understood `@join ... if` to be used for predicating on the iteration variable.